File: rxincrmark.lua

--local posix = require'posix'




test  = function(endpoint)
	i = 0
	while(i ~= endpoint)
	do
		i = i + 1
	end
end

ti = os.time()

test(0xffFFffFF)

ti = os.time() - ti

print(ti)

File: rxincrmark.py
from time import time

def get_ms():
    return time()*1000

def busy(iterations):
    it = 0
    while it < iterations:
        it = it + 1
    return

ti = get_ms()
busy(0xffFFffFF)
print( (get_ms() - ti)  )
